README.md

```diff
@@ -1613,7 +1616,9 @@ So this example...
 ```
 {
   "bytes_sent_str": "1024",
-  "bytes_sent_long": 1024
+  "bytes_sent_long": 1024,
+  "has_message": true,
+  "raw_message": { "type":"example", "msg":"example of json message with type" }
```

Comment:
Where do `type` and `msg` come from in this example?    I don't understand how `#asJson{%m}` turns into `{ "type":"example", "msg":"example of json message with type" }`